app/assets/src/components/common/MetadataManualInput.jsx

```diff
@@ -81,8 +77,17 @@ class MetadataManualInput extends React.Component {
       }
     });
 
-    // Default to the first host genome (Human)
-    this.props.samples.map(sample => this.updateHostGenome(1, sample));
+    // Set all to Human by default
```

Comment:
Maybe add a comment saying "we can't use updateMetadataField because we are updating a bunch of fields in bulk."